shared/pingpong/accounts.go

```diff
@@ -197,9 +197,9 @@ func (pps *WorkerState) ensureAccounts(ac *libgoal.Client) (err error) {
 				srcAcctPresent = true
 			}
 
-			ai, err := ac.AccountInformation(addr, true)
-			if err != nil {
-				return err
+			ai, aiErr := ac.AccountInformation(addr, true)
+			if aiErr != nil {
+				return aiErr
```

Comment:
```suggestion
			ai, err := ac.AccountInformation(addr, true)
			if err != nil {
				return
```
Suggestion to stay within style of the rest of this function. Alternately, we might want to root out all the naked returns -but that could be in a follow up as previously discussed.

Reply:
(the spacing is all messed up in the suggestion)